### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by vidyavemulapalli**
update to 'type' action to 'pressSequentially' as per new Playwright/test 1.38

### CODE REVIEW

File: docs/Utilities.md [hunk 1]
```diff
@@ -176,11 +176,11 @@ In this example, we're using various functions from `action-utils`:
 
 2. `click(input: string | Locator, options?: ClickOptions)`: This function clicks an element on the page. The input parameter is a string or Locator representing the element you want to click, and the options parameter is an optional parameter that specifies additional click options.
 
-3. `fill(input: string | Locator, value: string, options?: FillOptions)`: This function fills a form field with a specific value. The input parameter is a string or Locator representing the form field you want to fill, the value parameter is the value you want to fill the form field with, and the options parameter is an optional parameter that specifies additional fill options.
+3. `pressSequentially(input: string | Locator, value: string, options?: FillOptions)`: This function fills a form field with a specific value. The input parameter is a string or Locator representing the form field you want to fill, the value parameter is the value you want to fill the form field with, and the options parameter is an optional parameter that specifies additional fill options.
```
Comment (vasu31dev) [inline docs/Utilities.md:179]: @vidyavemulapalli fill is replaced with pressSequentially. Needs correction. Also Can you please add the below points too for fill?
- Clears input
- single input event

File: docs/Utilities.md [hunk 2]
```diff
@@ -176,11 +176,11 @@ In this example, we're using various functions from `action-utils`:
...
 
-4. `type(input: string | Locator, value: string, options?: TypeOptions)`: This function types into a field character by character, as if it was a user with a real keyboard. The input parameter is a string or Locator representing the form field you want to fill, the value parameter is the value you want to fill the form field with, and the options parameter is an optional parameter that specifies additional type options.
+4. `pressSequentially(input: string | Locator, value: string, options?: TypeOptions)`: This function enters text into a field character by character, as if it was a user with a real keyboard. The input parameter is a string or Locator representing the form field you want to enter the text, the value parameter is the value you want to enter the form field with, and the options parameter is an optional parameter that specifies additional PressSequentially options.
```
Comment (vasu31dev) [inline docs/Utilities.md:181]: @vidyavemulapalli Can you please add the below points too for pressSequentially?
- Does not Clears input
- sends a keydown, keypress/input, and keyup event for each character in the text

**Review by vasu31dev — CHANGES_REQUESTED** [review]
Please update the document as per the review comments

File: docs/Utilities.md (hunk 2 above)
Comment (vidyavemulapalli) [reply docs/Utilities.md]: @vasu31dev  Addressed the feedback comments in the latest commit

**Review by vasu31dev — APPROVED** [review]
Thanks for adding the feedback. looks good now!

**Pull request merged**